### ISSUE TITLE
False Positive: Unnecessary call arguments specification for array_keys

### ISSUE THREAD

**Issue by niconoe-** [kalessil/phpinspectionsea#1352]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended), 3.0.13 |
| Language level | PHP 7.2                         |

Current behaviour
--
When trying to list keys of an array where values are null, you might want to to this:
```php
$myNullKeys = array_keys($myArray, null);
```
But the inspection says the second parameter can be safely dropped. This is absolutely wrong because dropping this parameter will completly change the behavior of "array_keys".

Expected behaviour
--
Please advise to add the third parameter to true to make safer strict comparisons.

Thanks a lot for this great tool BTW!

**Comment by kalessil** [comment]
@niconoe- : you are welcome and thank you for reporting the issue =)

Unfortunately I can not reproduce the issue, would you please provide a simplified case for this issue?

**Comment by niconoe-** [comment]
@kalessil I just reproduiced the issue with the plugin at version 3.0.14 with PHPStorm 2019.2.

The issue is described like this:
![phpinspectionea-1352-1](https://user-images.githubusercontent.com/9560327/62539914-9e0f1e80-b856-11e9-9b6e-08d6023e4726.png)

You can try this snippet on every level (script level, namespaced script level, class-in method level namespaced class-in method level) and you'll get the inspection warning every time.

Here is the snippet you can see on my snapshot:
```php
$myArray = ['1' => null, 5 => null, 'foo' => 'bar', 'test' => null];
$myNullKeys = array_keys($myArray, null);
```
So you can copy/paste it and give it a try on your side.

Thanks a lot for your answer.

**Comment by kalessil** [comment]
Reproduced and fixed!

**Comment by niconoe-** [comment]
Awesome! Thank you!
